### ISSUE TITLE
[IDEA] Link with Nextcloud Notes

### ISSUE THREAD

**Issue by adocampo** [Zren/plasma-applet-todolist#2]
Hi there!
I like the applet! But, did you consider to make it able to connect to you own nextcloud instance? It will be AWESOME then! :D

**Comment by digitalethics** [comment]
@adocampo I think this is probably beyond the scope of this widget and would be better suited for standalone desktop and platform agnostic applications. For synchronization features you can try Joplin. Are you still using TodoList regularily?

**Comment by adocampo** [comment]
Not really. Without synchronization the tool is cool but it has no use for me.
Joplin is nice, but I'd rather prefer Carnet, I think is the best open source note taking alternative to Google Keep. It has room to improve, but it has a lot of potential. And it can sync with Nextlcloud and webdav servers.
